backends/advanced/init.py

```
@@ -0,0 +1,512 @@
+        if enable_https:
+            self.console.print("[blue][INFO][/blue] HTTPS enables microphone access in browsers")
+            self.console.print("[blue][INFO][/blue] For distributed deployments, use your Tailscale IP (e.g., 100.64.1.2)")
+            self.console.print("[blue][INFO][/blue] For local-only access, use 'localhost'")
+            server_ip = self.prompt_value("Server IP/Domain for SSL certificate (Tailscale IP or localhost)", "localhost")
+            
+            # Generate SSL certificates
+            self.console.print("[blue][INFO][/blue] Generating SSL certificates...")
+            ssl_script = Path("ssl/generate-ssl.sh")
+            if ssl_script.exists():
+                try:
+                    subprocess.run([str(ssl_script), server_ip], check=True, cwd=".")
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)